### PR TITLE
Jalmogo/refactor story navigator

### PR DESCRIPTION
**Pull request by jalMogo**
This PR:
 * renames StoryNavigator to FeaturedPlacesNavigator
 * refactors the data structures used to store FeaturedPlaces config
 * removes the "tranformStoryConfig" in our config-loader - it is no longer needed
 * reduces various other complexity by removing some features like having multiple FeaturedSite configs. We can only have one now.

Note that there is a pretty big diff because I ran an autoformatter over the json configs while editing.

### CODE REVIEW

File: src/base/static/components/organisms/content-panel.js [hunk 1]
```diff
@@ -56,7 +56,6 @@ const ContentPanelInnerContainer = styled("div")(props => ({
   }px 15px`,
   boxSizing: "border-box",
   scrollbarWidth: "none",
-  "-ms-overflow-style": "none",
```
Comment (jalMogo) [inline src/base/static/components/organisms/content-panel.js]: Note that this diff is unrelated to the PR.

I was getting this error in the browser:

>Using kebab-case for css properties in objects is not supported. Did you mean msOverflowStyle?

I also looked it up here:
https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-overflow-style

which says:

>This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

So I figure it's best to just delete it. @goldpbear Does that seem reasonable?

Comment (goldpbear) [reply src/base/static/components/organisms/content-panel.js]: I was seeing that too-- thanks for tracking it down. I think I added that as part of a set of properties design to hide scrollbars on some (though not all) browsers.

If it's not recommended for production, then I suppose removing it is the best option.
